app/src/examples/SharedStyleExample.tsx

```diff
@@ -12,7 +12,7 @@ export default function SharedStyleExample() {
 
   const [blueCounter, setBlueCounter] = useState<number>(0);
   const [greenCounter, setGreenCounter] = useState<number>(0);
-  const [itemList, setItemList] = useState<any>([]);
+  const [itemList, setItemList] = useState<JSX.Element[]>([]);
```

Comment:
is `JSX.Element` indeed a correct choice here? why not `React.Node`?

Reply:
[b3f0654](https://github.com/software-mansion/react-native-reanimated/pull/5361/commits/b3f065496ac61dc94b104961fa1f5d2098f2953a)